benchmark-framework/src/main/java/io/openmessaging/benchmark/WorkloadGenerator.java

```diff
@@ -76,10 +76,11 @@ public TestResult run() throws Exception {
         List<String> topics = worker.createTopics(new TopicsInfo(workload.topics, workload.partitionsPerTopic));
         log.info("Created {} topics in {} ms", topics.size(), timer.elapsedMillis());
 
-        createConsumers(topics);
         createProducers(topics);
+        createConsumers(topics);
 
-        ensureTopicsAreReady();
+	// Current versions of Kafka don't need to ensure topics are ready. Just create the producers first
+        // ensureTopicsAreReady();
```

Comment:
Why commenting this?

Reply:
It did not make a difference for me. I was having some trouble with the routine as it was hit or miss that it would work when testing the Confluent JMS Client

Reply:
Reverting that change is looking good. The troubles I had were with the Confluent JMS Client. I will turn this back on in the mentioned commit.